src/public/CowSwap.json

```
@@ -564,6 +564,14 @@
       "decimals": "18",
       "chainId": 1,
       "logoURI": "https://raw.githubusercontent.com/cowprotocol/token-lists/main/src/public/images/1/0xfcf8eda095e37a41e002e266daad7efc1579bc0a/logo.png"
+    },
+    {
+      "address": "0xe95a203b1a91a908f9b9ce46459d101078c2c3cb",
+      "symbol": "ankrETH",
+      "name": "Ankr Staked ETH",
+      "decimals": "18",
```

Manual bug fix (has been fixed for newer workflows)

```suggestion
      "decimals": 18,
```